src/App.tsx

```diff
@@ -65,7 +65,7 @@ export function App() {
 					{/* protected routes */}
 					<Route element={<ProtectRoute user={user} redirectPath="/" />}>
 						<Route
-							path="/list"
+							path="/list/:listName"
```

Comment:
One thing I noticed as a side effect is that the list option in the navigation bar now shows a "Page Not Found" error when clicked. The same happens when I click on View List button on the ManageList view. It might be happening due to this part, because we do not have a route defined for just /list anymore.

Reply:
Yeah I realized the side effect afterwards unfortunately. I'm exploring ways we can route to the list path directly in the view list navigation. I'm trying to see if I can import the object I created for list item to use it in the manage-list view. In terms of the list navigation directing to page not found, since, I'm still exploring that. Do we need the navigation to the \list navigation view anymore ? Unless there's other parts of the application that are impacted (looking into it), could we remove that navigation bar altogether since our lists are being directly accessed now through clicking them? @tannaurus @adiati98 @alex-andria

Reply:
I'm curious about the idea of placing all the singlelist elements into the list view altogether. Then once each list is clicked, it'll redirect to its own view?

Reply:
Whewww! So I was able to fix the issue of view list redirected to the 404 no page found since the \list view is out of commission for now. The name of the list is included in the end of the listPath parameter we are passing in the addItem form.  I extracted and stored the name of each list from the  listPath using the good ol - fashion pop() method! Using string interpolation, I appended our useNavigate in the addItem form  to include the name. Now we should successfully still be able to navigate to the list when clicking the "view list" button on the addItem form!

Reply:
if we end up wanting to keep the `/list` route we should just be able to have botht he one with the params and the basic list route as long as we order them properly. I beleive if we have the non param route first it would still work for both routes as we intend.

```
suggestion

	<Route
		path="/list"
		lement={<List data={data} listPath={listPath} />}
	/>						
    <Route
		path="/list/:listName"
		element={<List data={data} listPath={listPath} />}
	/>
```

Reply:
I'm noticing a 404 page when a new list is created:
![List](https://github.com/user-attachments/assets/61acfd0c-81b2-4f68-bd8e-4db170c71a6d)

Reply:
Towards @alex-andria 's observation, the same behavior happens when you try to navigate to the list via the `List` nav bar link. To fix that bug, we need to:
- Remove `List` from the nav bar (it no longer serves a purpose imo)
- Update `CreateList.handSubmit` to navigate to `/list:listName`

Reply:
i fixed the issue of the create list function leading to a 404 no page found. I wasn't able to navigate to /list:listName because  only because our List component route path is set to use the paramater :listName as the list name at the top of the page view and :listName appears at the top that way! I had to manipulate the "path" string to grab the name from it. 

 On the note of the "list" nav bar. It seems like the majority of the team is okay with removing the list nav bar button from the application since it no longer serves its purpose of navigating to the lists. I believe this is a separate issue, and am hesistant to remove it now unless everyone is okay with it at this point.

Reply:
@eternalmaha I'm personally gonna vote/say go ahead and remove it. I feel like this PR really progressed the functionallity of the list name button and made the nav bar list a bit redundant and I think it would be ok to do here as well.

Reply:
removed!!